src/OmniSharp.Abstractions/Logging/LoggerExtensions.cs

```diff
@@ -1,22 +0,0 @@
-﻿using System;
-
-namespace Microsoft.Extensions.Logging
-{
-    public static class LoggerExceptions
-    {
-        public static void LogDebug(this ILogger logger, Exception ex, string message, params object[] args)
-        {
-            logger.LogDebug(0, ex, message, args);
-        }
-
-        public static void LogError(this ILogger logger, Exception ex, string message, params object[] args)
-        {
-            logger.LogError(0, ex, message, args);
-        }
-
-        public static void LogTrace(this ILogger logger, Exception ex, string message, params object[] args)
-        {
-            logger.LogTrace(0, ex, message, args);
-        }
-    }
-}
```

Comment:
Cool! Are these now in the forked CommandLine package?

Reply:
they are finally part of `Microsoft.Extensions.Logging`, we can now use them since we moved from 1.1.0 to 2.1.1 https://github.com/aspnet/Logging/pull/392